### DISCUSSION TITLE
food ?  foodlist?

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#314] [Q&A] [answered]
想問一下各位大大這邊為什麼只需要food而不用像上面是foodlist?
![226174](https://user-images.githubusercontent.com/90717869/168760363-df0a5a67-666c-4181-b96a-cb6eb42770f8.jpg)

**Comment by NordicVikings** [comment, marked as answer]
我在寫的時候也在想這裏為甚麼可以這樣，所以就查了一下，順便做了圖片，我不確定我是不是對的，有可能是我想太多
(沒想到圖片這麼胡)
![image](https://user-images.githubusercontent.com/99714197/168767750-e19082ed-fe34-4940-a9e2-a2521b27c95a.png)
這是我查到的網站
https://blog.csdn.net/chenmozhe22/article/details/81434081?fbclid=IwAR0OfXsnVR181QNHVnTwmN86ktdFZLXkmiQvjXo3qNZKWjPwYIW-45UuHCU
另外我不是大大我很菜

**Comment by luo910227** [comment]
謝謝了解了，但應該是第n個資料對吧
